src/oscar/management/commands/oscar_fork_statics.py

```diff
@@ -30,7 +35,7 @@ def handle(self, *args, **options):
                 "The folder %s already exists - aborting!" % destination)
 
         source = os.path.realpath(
-            os.path.join(os.path.dirname(__file__), '../../static_src'))
+            os.path.join(os.path.dirname(__file__), '../../static'))
```

Comment:
@jwayodi `static/` contains only the compiled assets - not the original less files etc - and I think the latter is what a project would need? Previously both the original and compiled assets were here, but now they are separate. I think we need to copy `static_src` (and ensure it is in the pypi package).

Reply:
We discussed this and agreed that it is in fact the compiled statics that we need to fork here.